core/domain/prod_validation_jobs_one_off.py

```diff
@@ -596,7 +596,7 @@ def validate(cls, item):
 
         cls._validate_post_commit_status(item)
 
-        if 'question' in item.id or 'skill' in item.id:
+        if item.id[0:8] == 'question' or item.id[0:5] == 'skill':
```

Comment:
Hi @aks681 -- in a follow-up PR, could you please change this to use .startswith() instead? That would be conceptually cleaner than taking slices like this.

Thanks!

Reply:
Yeah, will do that.

Reply:
Fixed in #8363